clients/wallet/src/stellar_wallet.rs

```diff
@@ -223,6 +179,19 @@ impl StellarWallet {
 		Ok(TransactionsResponseIter { records, next_page, client: horizon_client })
 	}
 
+	///return balances for public key
```

Comment:
The comment is a bit misleading. 
I'd rephrase to: "Returns the balances of this wallet's Stellar account"

Reply:
We should also consider renaming the function to `get_balances` as the result is a vector